mod/missions/languages/en.php

```diff
@@ -83,7 +83,7 @@
     "missions:error:end_date_needs_input" =>    "Please enter an ideal completion date.",
     "missions:error:deadline_needs_input" =>    "Please enter an application deadline.",
     "missions:error:start_after_end" => "Your start date is after your end date.",
-    "missions:error:deadline_after_end" =>"Your start date is after your deadline.",
+    "missions:error:deadline_after_end" =>"Your start date is before your deadline.", 
```

Comment:
It's actually comparing the deadline and the end date, the start date has nothing to do with this one. I'm assuming it's just a copy/paste typo from way back when.

Reply:
```suggestion
    "missions:error:deadline_after_end" =>"Your end date is after your deadline.", 
```